foundations/installations/installations.md

```diff
@@ -8,7 +8,7 @@ If you are already using **MacOS**, **Ubuntu**, or [an official flavor of Ubuntu
 
 #### Be mindful of the OS you are using
 
-We can only support the operating systems indicated above. Our instructions have been tested with MacOS, Ubuntu, and official flavors of Ubuntu. We do not recommend installing an OS that is only based on Ubuntu (like Mint, Pop!_OS, ElementaryOS, etc).
+We can only support the operating systems indicated above. Our instructions have been tested with MacOS, Ubuntu, and official flavors of Ubuntu. We do not recommend installing an OS that is only based on Ubuntu (like Mint, Pop!\_OS, ElementaryOS, etc).
```

Comment:
Not sure we need to escape this underscore? The markdown parser on the site appears to handling it correctly already.

Reply:
The backslash seemed to have crept up due to Some sort of Auto Formatting , i disabled Format on save setting for this one , and it seems fine now.